src/app/components/query-explainer/query-explainer.types.ts

```diff
@@ -0,0 +1,41 @@
+export interface QueryExplainerResponse {
+    input: {
+        query: string;
+        queryDialect: string;
+        dataFormat: string;
+        schemaFormat: string;
+        include: string[];
+        skip: number;
+        limit: number;
+        datasets: [{ alias: string; blockHash: string; id: string }];
+    };
+    commitment: {
+        inputHash: string;
+        outputHash: string;
+        subQueriesHash: string;
+    };
+    proof: {
+        proofValue: string;
+        type: string;
+        verificationMethod: string;
```

Comment:
These look like enums too

Reply:
No, in this place string 


![image](https://github.com/user-attachments/assets/5312f07c-7810-429b-ba3d-f63e3c862dc6)